detectable/build.gradle

```diff
@@ -1,4 +1,6 @@
 dependencies {
+    implementation project(':configuration')
```

Comment:
I'd really prefer we didn't add configuration as a dependency to detectable - could we create an interface and hide FilterableEnumValue behind it? I'd rather detectables were immune to changes to detect configuration.

Reply:
Removed the new dependency by moving the FilterableEnumValue complexity to DetectableOptionFactory.